src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -4286,24 +4286,32 @@ public void Verify31958()
             var source = @"
 class C
 {
-    string? M(string s)
+    string? M1(string s)
     {
-        var d = (D)M;
+        var d = (D)M1; // 1
```

Comment:
To clarify why these tests changed: the nullable override analysis will refrain from warning about parameters if we have already warned about the return type. Thus to exercise all the same code paths in this test we need to have a method with an incompatible return type and another method with a compatible one.